asv_bench/benchmarks/groupby.py

```diff
@@ -510,6 +511,27 @@ def time_groupby_extra_cat_nosort(self):
         self.df_extra_cat.groupby("a", sort=False)["b"].count()
 
 
+class CategoricalFrame:
+    def setup(self):
```

Comment:
how is this different than the section right above?

Reply:
the values are categories in this new test

Reply:
can you add a comment to this class to that effect (and to the above that its just for grouping.